config_builder.go

```diff
@@ -1471,8 +1471,40 @@ func initNeutrinoBackend(ctx context.Context, cfg *Config, chainDir string,
 			lncfg.SqliteNeutrinoDBName, lncfg.NSNeutrinoDB,
 		)
 
+	case cfg.DB.Backend == kvdb.PostgresBackendName:
+		// This code needs to be in place because we did not start
+		// the postgres backend for neutrino at the beginning. Now we
+		// are also moving it into the postgres backend so we can phase
+		// out the bolt backend.
+		dbName := filepath.Join(dbPath, lncfg.NeutrinoDBName)
+		// Check if the file exists, because we cannot start a new
+		// neutrino db before remove the old data associated with it.
+		exists := fileExists(dbName)
+		if exists {
+			ltndLog.Infof("Neutrino Bolt DB found, removing it " +
+				"and starting fresh with postgres")
+
+			// Remove the block_headers.bin file
+			blockHeadersPath := filepath.Join(
+				dbPath, "block_headers.bin")
+			regFilterHeadersPath := filepath.Join(
+				dbPath, "reg_filter_headers.bin")
+
+			os.Remove(blockHeadersPath)
```

Comment:
Hmm, so we wouldn't migrate those DBs with `lndinit`? Since it's easier to just do a fresh sync?

Reply:
Nice catch, yes we need to make a distinction here, when the user who run this new code was already on sqlite-kv then we need to remove the data. I think it's way easier then to let him migrate the neutrino.db.

For the case where he used the migration tool to go to sql-kv and was running neutrino before, we should just ignore the db (if it is tombstoned) because the neutrino db will also be migrated.